modules/widgets/rsslinks-widget.php

```diff
@@ -141,12 +141,19 @@ function _rss_link( $type = 'posts', $args ) {
 
 		$link_item = '';
 		$format = $args['format'];
+		
+		if ( has_filter( 'jetpack_rsslinks_widget_target_blank' ) && apply_filters( 'jetpack_rsslinks_widget_target_blank', false ) ) {
```

Comment:
Why are we checking for whether it has filters, instead of just ... running the filter and seeing what comes up?  If it doesn't have anything, it shouldn't be any more expensive.